src/routes/+layout.svelte

```diff
@@ -49,9 +49,9 @@
 <AppShell>
 	<svelte:fragment slot="header">
 		<!-- App Bar -->
-		<AppBar>
+		<AppBar class="app-bar-custom">
```

Comment:
tbh, not really what this is / how it works

Reply:
It's just applying the `.app-bar-custom` class values (defined in `app.postcss`) to that element. 
It's a way to customize/ overwrite the default styling

Reply:
seems like it becomes a `<div>` in the DOM

<img width="869" alt="Screenshot 2024-12-03 at 14 29 35" src="https://github.com/user-attachments/assets/bff12096-d9e2-449e-9ca4-eb1bc3b2a416">